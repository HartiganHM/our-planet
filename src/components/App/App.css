.buffer {
  padding-top: 76px; }
