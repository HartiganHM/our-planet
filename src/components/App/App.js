import React from 'react';
import Header from '../../containers/Header/Header';
import Routes from '../Routes/Routes';
import './App.css';

const App = () => {
  return (
    <div className="App">
      <Header />
      <Routes />
    </div>
  );
};

export default App;
