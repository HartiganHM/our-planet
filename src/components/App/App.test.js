import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App tests', () => {
  it('Should match the snapshot', () => {
    const renderedApp = shallow(<App />);

    expect(renderedApp).toMatchSnapshot();
  });
});
