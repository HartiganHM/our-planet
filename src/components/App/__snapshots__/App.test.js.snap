// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests Should match the snapshot 1`] = `ShallowWrapper {}`;
